ngraph/core/CMakeLists.txt

```diff
@@ -72,6 +72,13 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
     set_target_properties(ngraph PROPERTIES LINK_FLAGS "/IGNORE:4217,4286")
 endif()
 
+set_source_files_properties("${CMAKE_CURRENT_SOURCE_DIR}/src/pass/convert_precision.cpp"
+                            "${CMAKE_CURRENT_SOURCE_DIR}/src/pass/convert_fp32_to_fp16.cpp"
+                            "${CMAKE_CURRENT_SOURCE_DIR}/src/op/type_relaxed.cpp"
+        PROPERTIES INCLUDE_DIRECTORIES "${CMAKE_CURRENT_SOURCE_DIR}/../../inference-engine/src/transformations/include/")
+
+target_compile_definitions(ngraph PRIVATE inference_engine_transformations_EXPORTS)
```

Comment:
Looks like we need to do the opposite thing - add `NGRAPH_API` to required entities inside transformation library

Reply:
Added.

Reply:
`target_compile_definitions(ngraph PRIVATE inference_engine_transformations_EXPORTS)` can now be removed since ngraph does not export anything which is marked as `TRANSFORMATIONS_API`